docker-compose.yml

```diff
@@ -42,9 +43,12 @@ services:
             POSTGRES_PASSWORD: password
             POSTGRES_USER: imageit
             POSTGRES_DB: imageit
-    gamma_backend:
-        build:
-            context: ./backend
-            dockerfile: dev.Dockerfile
+
+    redis:
+        image: bitnami/redis:5.0.8
```

Comment:
This is not the official Redis image. Any reason for this?

Reply:
Just because of the ENV feature that bitnami/redis gives